init.ts

```diff
@@ -92,12 +92,15 @@ setup(app, {
 	onInitHooks,
 })
 	.tap(createSuperuser)
-	.then(({ startServer, runFromCommandLine }) => {
-		if (process.argv.length > 2) {
-			return runFromCommandLine();
-		}
+	.then(({ startServer }) => {
 		return startServer(process.env.PORT || 1337).return();
 	})
+	.then(() => {
+		if (process.env.DEPLOYMENT === 'TEST') {
```

Comment:
```
function runFromMocha(context: AnyObject = global) {
	return ['afterEach', 'after', 'beforeEach', 'before', 'describe', 'it'].every(
		functionName => {
			return context[functionName] instanceof Function;
		},
	);
}
```
then...

```suggestion
		if (runFromMocha()) {
```

Reply:
@Page- @dfunckt what do you think? We can only run tests IF the process is started via Mocha rather than solely relying on an ENV var...

Reply:
We can stick the an ENV var for now